packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -83,6 +89,16 @@ export interface EngineReportingOptions {
   sendReportsImmediately?: boolean;
   // To remove the error message from traces, set this to true. Defaults to false
   maskErrorDetails?: boolean;
+  // Creates the client information attached to the traces sent to the Apollo
+  // backend
+  createClientInfo?: (
+    o: {
+      request: Request;
+      queryString?: string;
```

Comment:
I'm confused as to why we're passing the query information here? I can understand using the variables as well as query extensions, but I don't understand the query string.